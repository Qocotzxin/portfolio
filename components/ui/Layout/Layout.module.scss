.Layout {
  width: 100vw;
  height: 100vh;

  main {
    height: 100%;
    position: relative;
    z-index: -1;
  }

  &__transitionWrapper {
    padding-top: 4rem;
    padding-bottom: 4rem;
    height: 100%;
  }
}
