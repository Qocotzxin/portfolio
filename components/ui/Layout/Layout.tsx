import Navbar from "@ui/Navbar";
import Head from "next/head";
import { FC, PropsWithChildren } from "react";
import { Box, ColorModeScript } from "@chakra-ui/react";

const Home: FC<PropsWithChildren> = ({ children }) => {
  return (
    <Box
      w="100vw"
      h="100vh"
      bg="gray.50"
      _dark={{ bg: "gray.800" }}
      className="w-screen h-screen bg-slate-50 dark:bg-gray-900"
    >
      <Head>
        <title>Cristian Etchebarne</title>
        <meta charSet="utf-8" />
        <meta name="description" content="Cristian Etchebarne resume" />
        <link rel="icon" href="/favicon.ico" />
      </Head>

      <Navbar />

      <main>{children}</main>

      <footer></footer>
    </Box>
  );
};

export default Home;
