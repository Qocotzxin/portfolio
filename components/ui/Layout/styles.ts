import styles from "./Layout.module.scss";

export const Classes = {
  base: styles.Layout,
  transitionWrapper: styles.Layout__transitionWrapper,
};
