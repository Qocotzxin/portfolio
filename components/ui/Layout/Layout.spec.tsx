import { render } from "@testing-library/react";
import Layout from "./Layout";

describe("Layout.", () => {
  it("Should match snapshot.", () => {
    const { container } = render(<Layout />);
    expect(container).toMatchSnapshot();
  });
});
