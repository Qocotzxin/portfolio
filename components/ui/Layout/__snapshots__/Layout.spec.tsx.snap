// Vitest Snapshot v1

exports[`Layout. > Should match snapshot. 1`] = `
<div>
  <div
    class="w-screen h-screen bg-slate-50 dark:bg-gray-900 css-0"
  >
    <div
      class="css-0"
    >
      <a
        class="chakra-link css-0"
        href="/"
      />
      <div>
        <h1
          class="text-2xl font-body"
        >
          About
        </h1>
      </div>
    </div>
    <main />
    <footer />
  </div>
</div>
`;
